src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DefaultDocumentSnapshot.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Razor.ProjectSystem
+{
+    internal class DefaultDocumentSnapshot : DocumentSnapshot
```

Comment:
The idea is that we'd eventually implement enough stuff here to build a `RazorProjectFileSystem` based on what the project system knows about.